### DISCUSSION TITLE
How to publish own version of React Native fork from 0.71.12. Older documentation is not working anymore

### DISCUSSION THREAD

**Discussion by himanshuzeptonow** [react-native-community/discussions-and-proposals#697] [Ideas]
How to publish own version of React Native fork from 0.71.12. Older documentation is not working anymore.

**Comment by cortinico** [comment]
Which documentation are you referring to?

**Reply by himanshuzeptonow** [reply]
https://reactnative.dev/docs/0.60/publishing-forks

**Reply by himanshuzeptonow** [reply]
using same steps i was able to build fork from 0.68.x

**Reply by cortinico** [reply]
The closest documentation you can find is this one:
https://reactnative.dev/contributing/how-to-build-from-source

We don't support publishing forks and that's not recommended for general users.

**Reply by himanshuzeptonow** [reply]
tried above steps. but getting following error -> 

Project with path ':packages:react-native:ReactAndroid' not found in build 'react-native'.
